code/__DEFINES/language.dm

```diff
@@ -1,8 +1,9 @@
 #define NO_STUTTER 1
 #define TONGUELESS_SPEECH 2
-#define LANGUAGE_HIDE_ICON_IF_UNDERSTOOD 4
-#define LANGUAGE_HIDE_ICON_IF_NOT_UNDERSTOOD_WITH_LINGUIST_TRAIT 8 // even if you have linguist trait, you shouldn't see the icon
-#define LANGUAGE_ALWAYS_SHOW_ICON_IF_NOT_UNDERSTOOD 16
+#define LANGUAGE_HIDE_ICON_IF_YOU_UNDERSTAND 4
+#define LANGUAGE_HIDE_ICON_IF_YOU_SHOULD_NOT_RECOGNISE_WITH_LINGUIST_TRAIT 8 // even if you have linguist trait, you shouldn't see the icon
+#define LANGUAGE_ALWAYS_SHOW_ICON_IF_YOU_DO_NOT_UNDERSTAND 16
```

Comment:
This is kind of an unnecessary change. The defines were already pretty clear before, now they're just longer

The linguist one is particularly painful, please use comments instead
```suggestion
///Language icon will be hidden if you understand it, example: common
#define LANGUAGE_HIDE_ICON_IF_UNDERSTOOD 4
///Language icon will be hidden even if you have the linguist trait, example: agneusia 
#define LANGUAGE_ALWAYS_HIDE_ICON_IF_NOT_UNDERSTOOD 8
///Language icon will always be visible if you don't understand regardless of if you have the linguist trait, example: common
#define LANGUAGE_ALWAYS_SHOW_ICON_IF_NOT_UNDERSTOOD 16
//etc...
```
Something like this would be better
Now it's reasonably short and if someone doesn't understand how this flag is used they can just hover over it

Reply:
alright